src/jit/SimdX86Inl.h

```diff
@@ -765,6 +793,88 @@ static void simdEmitPMinMax(sljit_compiler* compiler, uint32_t operation, sljit_
     simdEmitSSEOp(compiler, is64 ? SimdOp::orpd : SimdOp::orps, rd, tmp);
 }
 
+static void simdEmitF64X2Max(sljit_compiler* compiler, sljit_s32 rd, sljit_s32 rn, sljit_s32 rm) {
+    sljit_s32 tmp = SLJIT_FR2;
```

Comment:
These 4 are quite similar, could we use some arguments instead of code duplication? When a few instructions change, they can be passed directly, or can be passed as flags:

https://github.com/Samsung/walrus/blob/jit_next/src/jit/SimdX86Inl.h#L357
https://github.com/Samsung/walrus/blob/jit_next/src/jit/SimdX86Inl.h#L879